src/Promitor.Agents.Scraper/Validation/Steps/Sinks/OpenTelemetryCollectorMetricSinkValidationStep.cs

```diff
@@ -33,6 +34,18 @@ public ValidationResult Run()
 
             var errorMessages = new List<string>();
             var collectorUri = openTelemetryCollectorSinkConfiguration.CollectorUri;
+            var collectorProtocol = openTelemetryCollectorSinkConfiguration.Collector.CollectorProtocol;
+
+            // Protocol Validation
+            if (string.IsNullOrWhiteSpace(collectorProtocol))
+            {
+                errorMessages.Add("No Protocol for the OpenTelemetry Collector is configured.");
+            }
+            else if (collectorProtocol != "http" && collectorProtocol != "grpc")
+            {
+                errorMessages.Add($"Configured Protocol ({collectorProtocol}) for the OpenTelemetry Collector should be either 'http' or 'grpc'.");
+            }
+            // URI Validation
             if (string.IsNullOrWhiteSpace(collectorUri))
```

Comment:
We need to include collector.Uri as well in this check.

Reply:
Check exists elsewhere I tihnk